### PR TITLE
(maint) Prepare for 0.99.5 release

### PR DESCRIPTION
**Pull request by demophoon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
...
 ## [Unreleased]
 
+## [0.99.5] - 2018-05-08
+### Added
+- Backported `ubership_light` task.
+
+### Changed
+- PROJECT_OVERRIDE can now be passed in to allow packaging to run on projects
+- which produce multiple package artifacts.
+  avoid name collisions with existing projects.
```
Comment (mwaggett) [inline CHANGELOG.md]: this looks.. amiss.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
...
+- Pkg::Tar now archives the working directory by default.
+- Updates to remove references to old stickler server.
+
+### Fixed
+- Packages promoted to a release branch no longer cause PEZ to fail
```
Comment (mwaggett) [inline CHANGELOG.md]: (nitpick) end with a .

**Comment by mwaggett** [comment]
We probably want the 0.66 in here? but I'm also fine letting that get caught in a future merge-up.

**Comment by demophoon** [comment]
@mwaggett What do you mean?

**Comment by mwaggett** [comment]
oops, left out a word - *the 0.66 changelog
So that this greater version changelog includes info about all previous releases.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.99.5] - 2018-05-08
+### Added
+- Backported `ubership_light` task.
```
[the remark this answers is not in the document]
Comment (mwaggett) [reply CHANGELOG.md]: also didn't notice in time for the 0.66 PR, but I think the task is actually called `uber_ship_lite`.

**Comment by demophoon** [comment]
ahh, i didn't realize the 0.6.x versions were below. Adding now.

**Comment by demophoon** [comment]
This should also be ready to go

File: CHANGELOG.md [hunk 4]
```diff
@@ -91,6 +103,16 @@ this is a final version.
...
 
+## [0.6.6] - 2018-05-08
+### Added
+- Backported `uber_ship_lite` task.
+
+### Changed
+- Pkg::Tar now archives the working directory by default.
+
+### Fixed
+- Packages promoted to a release branch no longer cause PEZ to fail
```
Comment (mwaggett) [inline CHANGELOG.md]: I think this should be in 0.99.5 as well.

**Pull request merged**